### PR TITLE
Discovery 9.1 docs

### PR DESCRIPTION
**Pull request by lzap**
I am late but here it is.

### CODE REVIEW

File: plugins/foreman_discovery/9.1/index.md [hunk 1]
```diff
@@ -0,0 +1,1353 @@
...
+    <td>{{page.imgver}}</td>
+    <td>{{page.cliver}}</td>
+  </tr>
+</table>
+
+### 1.1.1 Foreman Discovery plugin
+
+**9.1**: Release notes
+
+Mostly bugfix release with new support for notifications, possibility to add
```
Comment (mmoll) [inline plugins/foreman_discovery/9.1/index.md]: Mostly **a** bugfix release...

File: plugins/foreman_discovery/9.1/index.md [hunk 2]
```diff
@@ -0,0 +1,1353 @@
...
+arbitrary kexec kernel options, taxonomy for discovery rules and some UX
+improvements. The next major release will introduce many improvements in UX.
```
Comment (mmoll) [inline plugins/foreman_discovery/9.1/index.md]: I'm unsure if that should already be mentioned.

File: plugins/foreman_discovery/9.1/index.md [hunk 3]
```diff
@@ -0,0 +1,1353 @@
...
+
+We have released versions 9.0.0 and 9.1.0 but these were problematic with many
+bugs around manual discovery provisioning and Katello integration. All
+problems were finally solved in version 9.1.1. Make sure you upgrade directly
+to 9.1.1 when preparing production version bump.
```
Comment (mmoll) [inline plugins/foreman_discovery/9.1/index.md]: "...when updating production versions." sounds slightly better, IMHO, but I'm not a native speaker.

File: plugins/foreman_discovery/9.1/index.md [hunk 4]
```diff
@@ -0,0 +1,1353 @@
...
+
+Full changelog for the release:
+
+     Refs #20051 - mark compatible with Foreman 1.16+
```
Comment (mmoll) [inline plugins/foreman_discovery/9.1/index.md]: don't you want the links? :)

Comment (lzap) [reply plugins/foreman_discovery/9.1/index.md]: Not anymore :-)

**Comment by lzap** [comment]
Done!

File: plugins/foreman_discovery/9.1/index.md (hunk 4 above)
Comment (mmoll) [reply plugins/foreman_discovery/9.1/index.md]: mkay

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**